relay-cabi/src/processing.rs

```diff
@@ -137,7 +137,11 @@ pub unsafe extern "C" fn relay_store_normalizer_normalize_event(
         enable_trimming: config.enable_trimming.unwrap_or_default(),
         measurements: None,
     };
-    light_normalize_event(&mut event, light_normalization_config)?;
+    process_value(
+        &mut event,
+        &mut NormalizeProcessor::new(normalization_config),
+        ProcessingState::root(),
+    )?;
```

Comment:
So we still use store processor here after the new `NormalizeProcessor`?

Reply:
Yes. We still need to run both processors in some circumstances until we unify them.